crates/swc_ecma_transforms_base/src/resolver/mod.rs

```diff
@@ -920,6 +920,28 @@ impl VisitMut for Resolver<'_> {
         f.body.visit_mut_with(self);
     }
 
+    fn visit_mut_jsx_element_name(&mut self, node: &mut JSXElementName) {
+        if let JSXElementName::Ident(i) = node {
+            if i.as_ref().starts_with(|c: char| c.is_ascii_lowercase()) {
```

Comment:
```suggestion
            if i.as_ref().starts_with(char::is_ascii_lowercase) {
```

Reply:
We cannot do this because the original code implies auto dereferencing.

Reply:
👍 Oh I see